src/client/dfuse/ops/symlink.c

```diff
@@ -40,7 +40,7 @@ dfuse_cb_symlink(fuse_req_t req, const char *link,
 	DFUSE_TRA_UP(ie, parent, "inode");
 
 	rc = dfs_open(parent->ie_dfs->dfs_ns, parent->ie_obj, name,
-		      S_IFLNK, O_CREAT, 0, 0, link, &ie->ie_obj);
+		      S_IFLNK, O_CREAT | O_RDWR | O_EXCL, 0, link, &ie->ie_obj);
```

Comment:
Looks like a 0 was accidentally removed:
```suggestion
		      S_IFLNK, O_CREAT | O_RDWR | O_EXCL, 0, 0, link, &ie->ie_obj);
```

Reply:
Yes, thank you.  I clearly hadn't tested this and am working on a different branch now so hadn't had time to push the fix.